package.json

```diff
@@ -17,10 +17,10 @@
     "@emotion/react": "^11.13.0",
     "@emotion/styled": "^11.13.0",
     "@headlessui/react": "^1.7.14",
-    "@hyperlane-xyz/registry": "4.9.0",
-    "@hyperlane-xyz/sdk": "5.5.0",
-    "@hyperlane-xyz/utils": "5.5.0",
-    "@hyperlane-xyz/widgets": "5.5.0",
+    "@hyperlane-xyz/registry": "4.11.1",
+    "@hyperlane-xyz/sdk": "5.6.2",
+    "@hyperlane-xyz/utils": "5.6.2",
+    "@hyperlane-xyz/widgets": "5.6.2",
```

Comment:
Strictly speaking, 'functional' changes like dep upgrades should be done against `main` and then merged into the prod branches instead of directly to `nexus`. This is to minimize the divergence of the prod instances. But I understand this is easier

Reply:
makes sense - moving this into https://github.com/hyperlane-xyz/hyperlane-warp-ui-template/pull/304. I see there's a pretty big deviation between this branch and main apart from these though - out of caution I'll just update these deps

Reply:
Not certain what the process is for merging https://github.com/hyperlane-xyz/hyperlane-warp-ui-template/pull/279/ or what so just gonna merge this for now